polus-basic-flatfield-correction-plugin/src/main.py

```diff
@@ -41,8 +41,10 @@ def main():
 
     """ Get the input arguments """
     args = parser.parse_args()
-
     fpath = args.inpDir
+    """Checking if there is images subdirectory"""
+    if (os.path.isdir(Path(args.inpDir).joinpath('images'))):
```

Comment:
If you're loading the `Path` class, why use the `os` module? You can just use the `is_dir` method of the `Path` class. Also, the result of this if statement could cause `fpath` to change from a `str` to a `Path`. The class of `fpath` should remain consistent.